appinventor/appengine/build.xml

```diff
@@ -23,6 +23,7 @@
   <scriptdef name="generateguid" language="javascript">
     <attribute name="property" />
     <![CDATA[
+             load("nashorn:mozilla_compat.js");
```

Comment:
@josmas This throws an exception with Java 7. I've also found this solution, but it needs to be wrapped in a try-catch block.

-Jeff